### PR TITLE
Clueweb experiment updates

### PR DESCRIPTION
**Pull request by lintool**
After upgrading to Lucene 6.3.0, I reran ClueWeb09b and ClueWeb12-B13 experiments and updated the documentation.

### CODE REVIEW

File: docs/experiments-clueweb09b.md [hunk 1]
```diff
@@ -3,31 +3,39 @@
...
+After indexing is done, you should be able to perform a retrieval as follows:
 
 ```
-sh target/appassembler/bin/SearchWebCollection -topicreader Webxml -index lucene-index.cw09b.pos -bm25 \
+sh target/appassembler/bin/SearchWebCollection \
+  -topicreader Webxml -index lucene-index.cw09b.pos+docvectors -bm25 \
   -topics src/main/resources/topics-and-qrels/topics.web.51-100.txt -output run.web.51-100.bm25.txt
 ```
 
 You should then be able to evaluate using `trec_eval`, as with Gov2 above. With the topics and qrels in `src/main/resources/topics-and-qrels/`, you should be able to replicate the following results:
```
Comment (Peilin-Yang) [inline docs/experiments-clueweb09b.md]: shall we add the usage of `gdeval.pl` also?

File: docs/experiments-clueweb12-b13.md [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+After indexing is done, you should be able to perform a retrieval run:
 
 ```
-sh target/appassembler/bin/SearchWebCollection -topicreader Webxml -index lucene-index.cw12b13.pos -bm25 \
+sh target/appassembler/bin/SearchWebCollection \
+  -topicreader Webxml -index lucene-index.cw12b13.pos+docvectors -bm25 \
   -topics src/main/resources/topics-and-qrels/topics.web.201-250.txt -output run.web.201-250.bm25.txt
 ```
 
 You should then be able to evaluate using `trec_eval`, as with Gov2 and ClueWeb09 above. With the topics and qrels in `src/main/resources/topics-and-qrels/`, you should be able to replicate the following results:
```
Comment (Peilin-Yang) [inline docs/experiments-clueweb12-b13.md]: shall we add the usage of `gdeval.pl` also?

File: docs/experiments-clueweb09b.md (hunk 1 above)
Comment (lintool) [inline docs/experiments-clueweb09b.md]: okay

File: docs/experiments-clueweb12-b13.md (hunk 2 above)
Comment (lintool) [inline docs/experiments-clueweb12-b13.md]: okay

**Comment by lintool** [comment]
Added new results for *all* of ClueWeb12!

**Comment by Peilin-Yang** [comment]
ship it!

**Pull request merged**